frontend/app/routes/create-service.tsx

```
@@ -0,0 +1,26 @@
+import ServiceForm from "~/components/service-form";
+
+export function meta({}) {
```

Biome (1.9.4)

[error] Unexpected empty object pattern.



(lint/correctness/noEmptyPattern)